### ISSUE TITLE
unhandled error case

### ISSUE THREAD

**Issue by bAndie91** [apenwarr/gitbuilder#11]
https://github.com/apenwarr/gitbuilder/blob/bf70974a8e70b544b771b30b0849e12a398887ff/runlock#L42

hi i noticed it is not handled when `exec` does return. see perldoc:

> It fails and returns false only if the command does not exist

so NOTREACHED is actually reached sometime

cheers.

**Comment by apenwarr** [comment]
Yeah, that's a good point, though it doesn't come up very often I guess :)
